model/Core/Vocabularies/ExternalReferenceType.md

```diff
@@ -22,19 +22,34 @@ ExteralReferenceType specifies the type of an external reference.
 - buildMeta: A reference build metadata related to a published package.
 - buildSystem: A reference build system used to create or publish the package.
 - chat: A reference to the instant messaging system used by the maintainer for a package.
+- certificationReport: A reference to certification report for a package from an accredited/independent body.
+- componentAnalysisReport: A reference to a Software Component Analysis (SCA) report.
 - documentation: A reference to the documentation for a package.
+- dynamicAnalysisReport: A reference to dynamic analysis report for a package.
 - funding: A reference to funding informaton related to a package.
 - issueTracker: A reference to the issue tracker for a package.
 - mailingList: A reference to the mailing list used by the maintainer for a package.
 - metrics: A reference to metrics related to package such as OpenSSF scorecards.
 - license: A reference to additional license information related to an artifact.
 - other: Used when the type doesn't match any of the other options.
+- product: A reference to product information related to a package.
```

Comment:
This isn't clear to me - is this something that should be a shippable product,  or is it metadata about a product?